src/main/java/fileio_use_case/CalendarCourseBuilderInteractor.java

```diff
@@ -0,0 +1,25 @@
+package fileio_use_case;
+
+import entities.CalendarCourse;
+import entities.Section;
+
+import java.util.ArrayList;
+import java.util.List;
+/** Builds CalendarCourse **/
+public class CalendarCourseBuilderInteractor {
+    protected String title;
+    protected List<Section> sections;
+    protected String courseSession;
+    protected String courseCode;
+    protected String breadth;
+    public CalendarCourseBuilderInteractor(String title, ArrayList<Section> sections, String courseSession, String courseCode, String breadth) {
+        this.title = title;
+        this.sections = sections;
+        this.courseSession = courseSession;
+        this.courseCode = courseCode;
+        this.breadth = breadth;
+    }
+    public CalendarCourse newCourse(){
```

Comment:
Is this clean?

Reply:
Yes, I think since interactors can interact with entities